app/models/task.py

```diff
@@ -4,3 +4,8 @@
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable = True)
+
```

Comment:
We'll need to return task instances as dictionaries in our routes, so adding a helper method to do so will help us get rid of a lot of repetitive dictionary logic.

```
def to_dict(self):
    task = {
        "id": new_task.task_id,
        "title": new_task.title,
        "description": new_task.description,
        "is_complete": new_task.completed_at is not None
    }
    return task
```